modAionImpl/test/org/aion/zero/impl/cli/CliTest.java

```diff
@@ -603,6 +607,285 @@ public void testConfig_withMigration(String option) {
         Cli.copyRecursively(config, mainnetConfig);
     }
 
+    /** Parameters for testing {@Link #testPort(String[], ReturnType, String, String)}. */
+    @SuppressWarnings("unused")
+    private Object parametersWithPort() {
+        List<Object> parameters = new ArrayList<>();
+
+        String[] port_options = new String[] {"-p", "--port"};
+        String[] net_options = new String[] {"-n", "--network"};
+        String[] dir_options = new String[] {"-d", "--datadir"};
+        String expectedPath = MAIN_BASE_PATH.getAbsolutePath();
+        String expPathOnError = MAIN_BASE_PATH.getAbsolutePath();
+        String expPortOnError = Integer.toString(cfg.getNet().getP2p().getPort());
+
+        // port alone
+        for (String opPort : port_options) {
+            // without parameter
+            parameters.add(
+                    new Object[] {new String[] {opPort}, ERROR, expPathOnError, expPortOnError});
+            // with two parameters
+            parameters.add(
+                    new Object[] {
+                        new String[] {opPort, TEST_PORT, TEST_PORT},
+                        ERROR,
+                        expPathOnError,
+                        expPortOnError
+                    });
+            // with invalid parameter
+            parameters.add(
+                    new Object[] {
+                        new String[] {opPort, INVALID_PORT}, RUN, expPathOnError, expPortOnError
+                    });
+            parameters.add(
+                    new Object[] {
+                        new String[] {opPort, "-12345"}, RUN, expPathOnError, expPortOnError
+                    });
+            // with testing port number
```

Comment:
we also need here a string value for the port that is not a number, e.g. `test`

Reply:
added